src/main/java/core/basesyntax/state/FigureSupplier.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax.state;
+
+import core.basesyntax.figures.Circle;
+import core.basesyntax.figures.IsoscelesTrapezoid;
+import core.basesyntax.figures.Rectangle;
+import core.basesyntax.figures.RightTriangle;
+import core.basesyntax.figures.Square;
+
+public class FigureSupplier {
+    private static String []figureArr = {"Circle", "Isosceles Trapezoid", "Rectangle",
+            "Right Triangle", "Square"};
+
+    public static Figure getRandomFigure() {
+        int rndNum = (int)(Math.random() * figureArr.length);
```

Comment:
Don't abbreviate variable names by dropping letters from the middle of the word. It is not readable. Same in all places in you're code

Reply:
done